src/coreclr/jit/fgbasic.cpp

```diff
@@ -899,9 +936,24 @@ void Compiler::fgFindJumpTargets(const BYTE* codeAddr, IL_OFFSET codeSize, Fixed
 
     CORINFO_RESOLVED_TOKEN resolvedToken;
 
+    OPCODE opcode     = CEE_NOP;
+    OPCODE prevOpcode = CEE_NOP;
+    bool   handled    = false;
     while (codeAddr < codeEndp)
     {
-        OPCODE opcode = (OPCODE)getU1LittleEndian(codeAddr);
+        if (!handled && !conservativeInliner)
+        {
+            // Push SLOT_UNKNOWN to the stack since the current opcode was not handled
+            pushedStack.PushUnknown();
```

Comment:
This still leaves us with potentially bogus non-top stack values for operations that do multiple pops, right?

How hard would it be to get the stack model "correct"? 

For everything but calls we should be able to make it table driven as the information is in `opcode.def`.

Reply:
`PushUnknown` means we didn't find anything interesting for inliner in the previous opcode. I like the idea of making it table driven but in a separate PR if you don't mind. It works fine with the current implementation and doesn't change anything for the `DefaultPolicy` basically (when `preciseScan` is false)

Reply:
Doing it as a follow up is fine.